Client/Public/JS/Controllers/GameController.js

```diff
@@ -204,6 +217,60 @@ function outputToFlavorTextArea(text) {
     $(messageJustAppended).fadeIn(350);
 }
 
+function createTooltips() {
```

Comment:
NTS-4: Reorganize functions so they are in the proper place in the JavaScript in terms of execution.